dashboard/src/types/tree/TreeDetails.tsx

```diff
@@ -141,7 +141,16 @@ export const zPossibleValidator = z
   .enum(possibleTabs)
   .catch('treeDetails.boots');
 
-const possibleTableFilter = ['error', 'success', 'all'] as const;
+const possibleTableFilter = [
```

Comment:
we will porobably have to have `testsTableFilter, bootsTableFilter, buildsTableFilter`